lib/stanford-mods/origin_info.rb

```diff
@@ -14,29 +14,38 @@ class Record < ::Mods::Record
 
 # -- likely to be private or protected
 
-      # return all /originInfo/dateCreated elements in MODS records
-      # @return [Nokogiri::XML::NodeSet<Nokogiri::XML::Element>]
-      def date_created_nodeset
-        @mods_ng_xml.origin_info.dateCreated   # returns nokogiri element/node objects
-      end
-
-      # return all /originInfo/dateIssued elements in MODS records
-      # @return [Nokogiri::XML::NodeSet<Nokogiri::XML::Element>]
-      def date_issued_nodeset
-        @mods_ng_xml.origin_info.dateIssued   # returns nokogiri element/node objects
+      # return /originInfo/dateCreated elements in MODS records
+      # @param [Boolean] ignore_approximate true if approximate dates (per qualifier attribute)
+      #   should be excluded; false approximate dates should be included
+      # @return [Array<Nokogiri::XML::Element>]
+      def date_created_elements(ignore_approximate=false)
+        date_created_nodeset = @mods_ng_xml.origin_info.dateCreated
+        return self.class.remove_approximate(date_created_nodeset) if ignore_approximate
+        date_created_nodeset.to_a
```

Comment:
I am explicitly returning an Array now.  The old way returned a NodeSet, and when I added the ignore_approximate param, the method returned an Array if ignore_approximate was true, a NodeSet if ignore_approximate was false.   

So yes, the behavior is changed, which is why I changed all the comments and the calling methods.  :-)  This seemed the path of least resistance and cleanest code.